Pod/Classes/NSURLSession/Matchers/SimpleRequestMatcher.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  SimpleRequestMatcher.swift
+//  Pods
+//
+//  Created by Sam Dean on 02/02/2016.
+//
+//
+
+import Foundation
+
+/**
+ This struct matches by matching the path to a regular expression and by 
+ comparing the methods for equality.
+*/
+struct SimpleRequestMatcher : RequestMatcher {
+    
+    let pathMatcher: NSRegularExpression
+    
+    let method: String
+    
+    init(url: NSURL, method: String) {
```

Comment:
I wasn't sure - in the end I assumed that if a string has been escaped by NSRegularExpression itself, then it's probably fine?